lib/src/supertokens_dio_extension.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:dio/dio.dart';
+import 'package:supertokens_flutter/src/dio-interceptor-wrapper.dart';
+
+/// Mixin for easy Dio instance setup.
+///
+/// Usage:
+/// ```dart
+///
+/// final dio = Dio()
+///   ..addSupertokensInterceptor()
+///   ..addSentry()
+///   // ...
+/// ```
+extension SuperTokensDioExtension on Dio {
```

Comment:
please add some tests that adds interceptors using this method.

Reply:
Ok, no problem

Reply:
Added a similar test as in `dioInterceptor_test.dart`. It only differs in the way dio is being set up, you can see [here](https://github.com/supertokens/supertokens-flutter/blob/f2e71038e3f6b2a6b6eb976a03369be1d0ec03c6/test/dio_extension_interceptor_test.dart)

All tests passed:
![image](https://github.com/supertokens/supertokens-flutter/assets/74528634/bbd6f591-6aad-4429-b09b-029f676ab6ea)

Reply:
I think you should add the node.js test server version requirements to CONTRIBUTION.md, cause I'm having trouble running tests on 18.13 node.js. On node.js 16.20.2 tests work fine. Or maybe i missed this requirements for nodejs somethere? :)